### PR TITLE
Update bower components

### PR DESCRIPTION
**Pull request by ctwoolsey**

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -1,23 +1,24 @@
...
-    "angular": "1.5.9",
-    "bootstrap-sass-official": "3.2.0",
-    "angular-resource": "1.5.9",
-    "angular-sanitize": "1.3.0",
-    "angular-touch": "1.3.0",
-    "angular-ui-router": "0.2.15",
-    "angularjs-toaster": "0.4.15",
-    "angular-bootstrap": "1.3.2",
-    "angular-ui-select": "0.19.4",
+    "angular": "^1.6.1",
```
Comment (abobwhite) [inline bower.json]: Please use exact version - no "~" or "^". This is how we got bit in the past.

File: bower.json [hunk 2]
```diff
@@ -1,23 +1,24 @@
...
+    "angular-ui-router": "0.4.2",
+    "angularjs-toaster": "2.0.0",
+    "angular-bootstrap": "2.5.0",
+    "angular-ui-select": "0.19.6",
     "jquery": "2.2.4",
     "moment": "2.14.1",
     "leaflet": "1.0.2",
-    "angular-simple-logger": "0.1.7"
+    "angular-simple-logger": "0.1.7",
+    "angular-animate": "^1.6.1"
```
Comment (abobwhite) [inline bower.json]: Same here

File: bower.json [hunk 3]
```diff
@@ -31,6 +32,7 @@
     }
   },
   "resolutions": {
-    "angular": "1.5.9"
+    "angular": "1.6.1",
+    "angular-ui-select": "^0.19.6"
```
Comment (abobwhite) [inline bower.json]: And here

**Comment by ctwoolsey** [comment]
Closing for now.  Updating created some unit test errors that need to be examined.

**Pull request closed without merging**